### PR TITLE
all: update cloud.google.com/go to 0.38.0

### PR DESCRIPTION
**Pull request by jba**
-

### CODE REVIEW

**Comment by eliben** [comment]
Sync the PR past https://github.com/google/go-cloud/commit/74dfb2d2ea5301cfa7e6930d26eb527029f6c874

**Comment by jba** [comment]
> Does this bring in the faster httpreplay :) ?

Not yet. That's at master. Things really got messy when I did that, so I'm taking it step by step.

File: go.mod [hunk 1]
```diff
@@ -28,10 +28,17 @@ require (
 	github.com/Azure/go-autorest/tracing v0.1.0 // indirect
 	github.com/GoogleCloudPlatform/cloudsql-proxy v0.0.0-20190418212003-6ac0b49e7197
 	github.com/Shopify/sarama v1.19.0
+	github.com/Shopify/toxiproxy v2.1.4+incompatible // indirect
 	github.com/aws/aws-sdk-go v1.19.16
+	github.com/coreos/bbolt v1.3.2 // indirect
```
Comment (eliben) [inline go.mod]: Any idea where this is coming from now? What does go mod why -m say?

Comment (jba) [reply go.mod]:
```
gocloud.dev/runtimevar/etcdvar
go.etcd.io/etcd/clientv3
go.etcd.io/etcd/clientv3.test
github.com/coreos/etcd/auth
github.com/coreos/etcd/mvcc/backend
github.com/coreos/bbolt
```

Comment (eliben) [reply go.mod]: Ugh, but your PR doesn't even update the etcd version. What a mess :/

**Pull request merged**